manim/camera/camera.py

```diff
@@ -48,22 +50,22 @@ class Camera:
 
     def __init__(
         self,
-        background_image=None,
-        frame_center=ORIGIN,
-        image_mode="RGBA",
-        n_channels=4,
-        pixel_array_dtype="uint8",
-        z_buff_func=lambda m: np.round(m.get_center()[2], 2),
-        cairo_line_width_multiple=0.01,
-        use_z_index=True,
-        background=None,
-        pixel_height=None,
-        pixel_width=None,
-        frame_height=None,
-        frame_width=None,
-        frame_rate=None,
+        background_image: Image = None,
+        frame_center: str = ORIGIN,
```

Comment:
No, frame_center is not a string,it is a coordinate. It might be `Sequence[float]`, but see my commend above.